chromium_src/chrome/browser/resources/side_panel/customize_chrome/customize_toolbar/toolbar.html.ts.lit_mangler.ts

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import { mangle } from 'lit_mangler'
+
+// Remove #miniToolbarBackground
+mangle(
+  (element: DocumentFragment) => {
+    // Remove the mini toolbar background.
+    const miniToolbarBackground = element.querySelector(
+      '#miniToolbarBackground',
+    )
+    if (!miniToolbarBackground) {
+      throw new Error(
+        '[Customize Chrome > Toolbar] #miniToolbarBackground is gone.',
+      )
+    }
+
+    miniToolbarBackground.remove()
+  },
+  (template) => template.text.includes('id="miniToolbarBackground"'),
+)
+
+// Remove #tipCard
+mangle(
+  (element: DocumentFragment) => {
+    // Remove the tip card.
+    const tipCard = element.querySelector('#tipCard')
+    if (!tipCard) {
+      throw new Error('[Customize Chrome > Toolbar] #tipCard is gone.')
+    }
+
+    tipCard.remove()
+  },
+  (template) => template.text.includes('id="tipCard"'),
+)
+
+// Move .intro-text out of the sp-card and insert it after the sp-card, wrapping
+// it with a new sp-card. The first sp-card is the one with the heading.
+mangle(
+  (element: DocumentFragment) => {
+    const introText = element.querySelector('.intro-text')
+    if (!introText) {
+      throw new Error('[Customize Chrome > Toolbar] .intro-text is gone.')
+    }
+
+    // Remove the intro text from its current position.
+    const firstSpCard = element.querySelector('.sp-card')
+    if (!firstSpCard) {
+      throw new Error('[Customize Chrome > Toolbar] .sp-card is gone.')
+    }
+    firstSpCard.removeChild(introText)
+
+    // As we are inserting the bundled HTML at build time, we don't have to sanitize the element.
+    // eslint-disable-next-line no-unsanitized/method
```

Comment:
@fallaciousreasoning , I'm wondering if I understand this correctly.

Reply:
That's correct, yeah :smile: